frontend/src/PostContentEditor.jsx

```diff
@@ -24,14 +24,18 @@ class MyEditor extends React.Component {
 
   constructor(props) {
     super(props);
+    const { setTextFormatted } = this.props;
```

Comment:
Use prop validation! Check out my quick overview of prop validation here (after the "Edit 3/28"): https://github.com/JumboCode/ProjectShare/issues/143

It's basically just to make sure that the data types of prop variables are what is expected, since Javascript isn't a typed language the way C/C++ are, for example.